.github/workflows/workflow.yml

```diff
@@ -93,10 +93,7 @@ jobs:
 
       # dune doesn't have any additional dependencies so we can build it right
       # away this makes it possible to see build errors as soon as possible
-      - run: opam exec -- make _boot/dune.exe
```

Comment:
Good to see another path eliminated from the yaml file. These are always a pain to keep in sync